src/mmw/js/src/modeling/tests.js

```diff
@@ -834,7 +834,8 @@ describe('Modeling', function() {
                 });
 
                 function fetchResultsAssertions(self) {
-                    assert(inputModSpy.calledTwice, 'updateInputModHash should have been called twice');
+                    // TODO: Re-enable tests https://github.com/WikiWatershed/model-my-watershed/issues/3442
```

Comment:
These tests are flaky and will often fail. Disabling them for now, to be re-enabled in #3442.